cpp/log/sqlite_db-inl.h

```diff
@@ -17,8 +17,26 @@
 // new node, it would do an initial load of its local database with
 // "synchronous" set to OFF, then put it back before starting normal
 // operation.
-DEFINE_bool(sqlite_synchronous_off, false,
-            "whether to set SQLite's \"synchronous\" to OFF");
+DEFINE_string(sqlite_synchronous_mode, "FULL",
+              "While SQLite synchronous option to use");
+
+DEFINE_string(sqlite_journal_mode, "WAL",
+              "Which SQLite journal_mode option to use.");
+
+DEFINE_string(sqlite_temp_store, "DISK",
+              "Which SQLite temp_store option to use.");
+
+DEFINE_int32(sqlite_cache_size, 100000,
+             "Number of 1KB btree pages to keep in memory.");
+
+DEFINE_bool(sqlite_batch_into_transactions, true,
+            "Whether to batch operations into transactions behind the scenes. "
+            "Transactions will always be committed when new STHs are written. "
+            "It's recommended to run with the FULL synchronous mode when "
```

Comment:
I'm not sure this recommendation makes much sense? I think the synchronization mode only changes what happens when a transaction is completed, so it doesn't really matter what it does, it'll just be happening less often.